core/templates/dev/head/domain/skill/SkillValidationService.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to validate a skill.
+ */
+
+oppia.factory('SkillValidationService', [
+  'ValidatorsService',
+  function(
+      ValidatorsService) {
+    var _isValidDescription = function(description) {
+      var allowDescriptionToBeBlank = false;
```

Comment:
I think it's fine to inline true/false on line 27 if you like. That said I quite like how this makes the meaning of the booleans more readable, so I'm fine with how you have it, too.

Reply:
left as-is